pkgs/development/tools/build-managers/wafHook/default.nix

```diff
@@ -1,7 +1,6 @@
-{ lib, stdenv, pkgs, python, makeSetupHook, waf }:
+{ lib, stdenv, pkgs, makeSetupHook, waf }:
 
 makeSetupHook {
-  deps = [ python ];
```

Comment:
Something to mention in changelogs?

Reply:
Good idea, but I'm going to target staging soon and I was told that PRs to staging should not include changelog to prevent annoying conflicts

Reply:
Ok. I would just mention that there how to fix wafHook usage by also include python in `nativeBuildInputs`.